### PR TITLE
update doc about vschema

### PR DESCRIPTION
**Pull request by sougou**
Signed-off-by: Sugu Sougoumarane <ssougou@gmail.com>

### CODE REVIEW

**Comment by demmer** [comment]
Ideally we should also include some description of the vschema ddl syntax.

File: content/docs/schema-management/mysql-schema.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+affects too many rows of a table, it will reject it.
+
+However, the down side is that it is a little too strict, and may not work for all use cases.
+
+## VTGate
+
+You can send a DDL directly to a VTGate just like you would send to a mysql instance. If the target is a sharded keyspace,
+then the DDL would be sprayed to all shards.
+
+If a specific shard fails you can target it directly using the `keyspace/<shard` syntax to retry the apply just to that shard.
```
Comment (deepthi) [inline content/docs/schema-management/mysql-schema.md]:
```suggestion
If a specific shard fails you can target it directly using the `keyspace/<shard>` syntax to retry the apply just to that shard.
```

File: content/docs/schema-management/routing-rules.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+In the above case, we are declaring that the `product` table is present in both `lookup` and `user`. If a query is issued
+using the unqualified `product` table, then VTGate will consider sending the query to both `lookup.product` as well
+as `user.uproduct` (note the name change).
+
+For example, if `user` was a sharded keyspace, and the query joined a `user` table with `product`, then vtgate will
+know that it's better to send the query to the `user` keyspace instead of `lookup`.
+
+Typically, table equivalence makes sense when a view table is materialized from a source table using VReplication.
+
+### Orhtogonality
```
Comment (deepthi) [inline content/docs/schema-management/routing-rules.md]:
```suggestion
### Orthogonality
```

**Review by deepthi — APPROVED** [review]
Looks good. Just a couple of typos.

**Comment by sougou** [comment]
@demmer I'm going to merge this for tomorrow's session. I'll add the DDL syntax as a separate PR.

**Pull request merged**